gtsam/linear/HessianFactor.cpp

```diff
@@ -470,7 +470,7 @@ std::shared_ptr<GaussianConditional> HessianFactor::eliminateCholesky(const Orde
 
     // TODO(frank): pre-allocate GaussianConditional and write into it
     const VerticalBlockMatrix Ab = info_.split(nFrontals);
-    conditional = std::make_shared<GaussianConditional>(keys_, nFrontals, Ab);
+    conditional = std::make_shared<GaussianConditional>(keys_, nFrontals, std::move(Ab));
```

Comment:
This is called *many* times. I had hoped this would make a performance difference.

Reply:
`Ab` is `const` and moving it has no effect.

Reply:
Wow, ok. Let me see whether that does anything.